cmd/antrea-agent/options_windows.go

```diff
@@ -19,13 +19,18 @@ package main
 import (
 	"fmt"
 	"strings"
+	"os/exec"
+
 
 	"k8s.io/component-base/featuregate"
 	"k8s.io/klog/v2"
 
 	"antrea.io/antrea/pkg/agent/config"
 	"antrea.io/antrea/pkg/features"
 	"antrea.io/antrea/pkg/ovs/ovsconfig"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
```

Comment:
The imports are grouped in this way: built-in, 3rd parties, antrea's own. Could you move them to the second group?

Reply:
Roger that :)